deployments/helm/nvidia-device-plugin/templates/_helpers.tpl

```diff
@@ -217,3 +217,15 @@ Pod annotations for the plugin and GFD
 {{- end -}}
 {{- toYaml $annotations }}
 {{- end -}}
+
+
+{{/*
+Collection of typed options for the device plugin.
+
+We convert this to JSON so that it can be included and converted to an object using fromJson.
+*/}}
+{{- define "nvidia-device-plugin.options" -}}
```

Comment:
Why can't this function return the typed options object instead of the Json? It seems a bit redundant the current way since we seem to couple all the calls to `nvidia-device-plugins.options` with `fromJson`

Reply:
I think it could and I will spend some time checking out how this should be done.

For what it's worth this is from: https://github.com/helm/helm/issues/11231#issuecomment-1234506859

Reply:
Understood, thanks for the context